rpcs3/Emu/RSX/VK/VKHelpers.h

```diff
@@ -754,7 +754,7 @@ namespace vk
 					return driver_vendor::AMD;
 				}
 
-				if (gpu_name.find("NVIDIA") != umax || gpu_name.find("GeForce") != umax)
+				if (gpu_name.find("NVIDIA") != umax || gpu_name.find("GeForce") != umax || gpu_name.find("Quadro") != umax)
```

Comment:
Don't quadro's have "NVIDIA" in the device string? I added "GeForce" check for the theoretical nouveau vulkan driver that will likely never see the light of day.

Reply:
It's "Quadro P500"